server.py

```diff
@@ -101,7 +103,7 @@ def getOfficers():
   """
   API function to retrieve officers
   Parameters:
-  /api/officers?position=[PRESIDENT,VICE_PRESIDENT,TREASURER]
+  /api/officers?position=[PRESIDENT,VICE_PRESIDENT,TREASURER,SECRETARY]
```

Comment:
Make sure this is consistent with what's on the enum (currently missing `TECHINCAL_OFFICER`)